scripts/check-and-run-apollo-codegen.sh

```diff
@@ -1,10 +1,31 @@
+# Only major and minor version should be specified here
 REQUIRED_APOLLO_CODEGEN_VERSION=0.11
 
+REQUIRED_APOLLO_CODEGEN_SEMVER="$REQUIRED_APOLLO_CODEGEN_VERSION.x"
```

Comment:
I don't think this is actually needed. You can run `npm install` with just the major and minor versions, and it will automatically install the latest patch version.